api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -622,7 +625,13 @@ def update_venue(venue_id: int) -> utils.BackofficeResponse:
 
     try:
         offerers_api.update_venue(
-            venue, author=current_user, contact_data=contact_data, criteria=criteria, admin_update=True, **attrs
+            venue,
+            author=current_user,
+            contact_data=contact_data,
+            criteria=criteria,
+            admin_update=True,
+            external_accessibility_url=form.acceslibre_url.data if isinstance(form, forms.EditVenueForm) else None,
```

Comment:
Il semble qu'on utilise plutôt `hasattr` dans cette fonction, je propose pour rester homogène : 
```suggestion
            external_accessibility_url=form.acceslibre_url.data if hasattr(form, "acceslibre_url") else None,
```